sim/druid/runes.go

```diff
@@ -69,7 +69,7 @@ func (druid *Druid) applySunfire() {
 			NumberOfTicks: ticks,
 			TickLength:    time.Second * 3,
 			OnSnapshot: func(sim *core.Simulation, target *core.Unit, dot *core.Dot, _ bool) {
-				dot.SnapshotBaseDamage = (baseDotDamage / float64(ticks)) + 0.13*dot.Spell.SpellPower()
+				dot.SnapshotBaseDamage = baseDotDamage + 0.13*dot.Spell.SpellPower()
```

Comment:
The base damage is actually `baseCalc * .65 * 4` but up above we only use `baseCalc * .65`. This means each tick was actually dealing 1/4 the correct damage. I opted to remove the division by # of ticks rather than multiply the base dot damage and then divide here

<img width="325" alt="image" src="https://github.com/wowsims/sod/assets/12898988/090ba985-5d92-404f-bfdc-bd9aeac5f26f">